### PR TITLE
ROX-19203: Add info for product usage page

### PR DESCRIPTION
**Pull request by kcarmichael08**
Version(s):

4.2+

[Issue](https://issues.redhat.com/browse/ROX-19203)

[Link to docs preview
](https://file.rdu.redhat.com/kcarmich/ROX-19203-product-usage/operating/use-system-health-dashboard.html#viewing-product-usage_use-system-health-dashboard)

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/viewing-product-usage.adoc [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+====
+The data shown is not sent to Red Hat or displayed as Prometheus metrics.
+====
+
+.Procedure
+
+. In the {product-title-short} portal, navigate to *Platform Configuration* -> *System Health*.
+. Click *Show product usage*.
+. In the *Start date* and *End date* fields, choose the dates for which you want to display data. This range is inclusive and depends on your time zone.
+. Optional: To download the data, click *Download CSV*.
```
Comment (0x656b694d) [inline modules/viewing-product-usage.adoc:37]:
```suggestion
. Optional: To download the detailed data, click *Download CSV*.
```

**Review by parametalol — CHANGES_REQUESTED** [review]

File: modules/viewing-product-usage.adoc [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+{ocp} provides its own usage reports; this information is intended for use with self-managed Kubernetes systems.
+====
+
+This page provides the following data:
+
+* Currently secured CPU units: The number of virtual CPUs used by your {product-title-short} secured clusters as of the latest metrics received from sensors.
+* Currently secured node count: The number of Kubernetes nodes secured by {product-title-short} as of the latest metrics received from sensors.
+* Maximum secured CPU units: The maximum number of virtual CPUs used by your {product-title-short} secured clusters for the time period as measured by the *Start date* and *End date*. This measurement only includes clusters that are still connected.
+* Maximum secured node count: The maximum number of Kubernetes nodes secured by {product-title-short} as as measured by the *Start date* and *End date*. This measurement only includes clusters that are still connected.
+* CPU units observation date: The date on which the data displayed on this page for the currently secured CPU units was collected.
```
[the remark this answers is not in the document]
Comment (kcarmichael08) [reply modules/viewing-product-usage.adoc:22]: We follow IBM Style:

 data - noun
Use as a singular noun with a singular verb, even though data is the plural of datum.

File: modules/viewing-product-usage.adoc [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+[NOTE]
+====
+{ocp} provides its own usage reports; this information is intended for use with self-managed Kubernetes systems.
+====
+
+This page provides the following data:
+
+* Currently secured CPU units: The number of virtual CPUs used by your {product-title-short} secured clusters as of the latest metrics received from sensors.
+* Currently secured node count: The number of Kubernetes nodes secured by {product-title-short} as of the latest metrics received from sensors.
+* Maximum secured CPU units: The maximum number of virtual CPUs used by your {product-title-short} secured clusters for the time period as measured by the *Start date* and *End date*. This measurement only includes clusters that are still connected.
```
[the remark this answers is not in the document]
Comment (kcarmichael08) [reply modules/viewing-product-usage.adoc]: I rewrote this a little based on the additional info you provided in slack which helped me understand better what this is measuring. Please lmk if it's OK or not.

**Review by parametalol — APPROVED** [review]
Thank you!

File: modules/viewing-product-usage.adoc [hunk 4]
```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * operating/use-system-health-dashboard.adoc
+:_content-type: PROCEDURE
+[id="viewing-product-usage_{context}"]
+= Viewing product usage data
+
+[role="_abstract"]
+{product-title-short} provides product usage data for the number of secured Kubernetes nodes and CPU units for secured clusters based on metrics collected from {product-title-short} sensors. This information can be useful in estimating {product-title-short} consumption data for reporting.
```
Comment (mburke5678) [inline modules/viewing-product-usage.adoc:9]:
```suggestion
{product-title-short} provides product usage data for the number of secured Kubernetes nodes and CPU units for secured clusters based on metrics collected from {product-title-short} sensors. This information can be useful to estimate {product-title-short} consumption data for reporting.
```

File: modules/viewing-product-usage.adoc [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
+For more information on how CPU units are defined in Kubernetes, see link:https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#meaning-of-cpu[CPU resource units].
+
+[NOTE]
+====
+{ocp} provides its own usage reports; this information is intended for use with self-managed Kubernetes systems.
+====
+
+This page provides the following data:
```
Comment (mburke5678) [inline modules/viewing-product-usage.adoc:18]: Which page?

File: modules/viewing-product-usage.adoc [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+
+* Currently secured CPU units: The number of Kubernetes CPU units used by your {product-title-short} secured clusters as of the latest metrics received from sensors.
+* Currently secured node count: The number of Kubernetes nodes secured by {product-title-short} as of the latest metrics received from sensors.
+* Maximum secured CPU units: The maximum number of CPU units used by your {product-title-short} secured clusters as measured hourly and aggregated for the time period of the *Start date* and *End date*.
+* Maximum secured node count: The maximum number of Kubernetes nodes secured by {product-title-short} as measured hourly and aggregated for the time period of the *Start date* and *End date*.
+* CPU units observation date: The date on which the data displayed on this page for the maximum secured CPU units was collected.
```
Comment (mburke5678) [inline modules/viewing-product-usage.adoc:24]:
```suggestion
* CPU units observation date: The date on which the maximum secured CPU units data was collected.
```

File: modules/viewing-product-usage.adoc [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+* Node count observation date: The date on which the data displayed on this page for the maximum secured node count was collected.
```
Comment (mburke5678) [inline modules/viewing-product-usage.adoc:25]:
```suggestion
* Node count observation date: The date on which the maximum secured node count data was collected.
```

File: modules/viewing-product-usage.adoc [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+
+[NOTE]
+====
+{ocp} provides its own usage reports; this information is intended for use with self-managed Kubernetes systems.
+====
+
+This page provides the following data:
+
+* Currently secured CPU units: The number of Kubernetes CPU units used by your {product-title-short} secured clusters as of the latest metrics received from sensors.
+* Currently secured node count: The number of Kubernetes nodes secured by {product-title-short} as of the latest metrics received from sensors.
```
Comment (mburke5678) [inline modules/viewing-product-usage.adoc:21]:
```suggestion
* Currently secured node count: The number of Kubernetes nodes secured by {product-title-short} as of the latest metrics collection.
```

File: modules/viewing-product-usage.adoc [hunk 9]
```diff
@@ -0,0 +1,45 @@
...
+* Maximum secured CPU units: The maximum number of CPU units used by your {product-title-short} secured clusters as measured hourly and aggregated for the time period of the *Start date* and *End date*.
```
Comment (mburke5678) [inline modules/viewing-product-usage.adoc:22]:
```suggestion
* Maximum secured CPU units: The maximum number of CPU units used by your {product-title-short} secured clusters, as measured hourly and aggregated for the time period defined by the *Start date* and *End date*.
```

File: modules/viewing-product-usage.adoc [hunk 10]
```diff
@@ -0,0 +1,45 @@
...
+* Maximum secured node count: The maximum number of Kubernetes nodes secured by {product-title-short} as measured hourly and aggregated for the time period of the *Start date* and *End date*.
```
Comment (mburke5678) [inline modules/viewing-product-usage.adoc:23]:
```suggestion
* Maximum secured node count: The maximum number of Kubernetes nodes secured by {product-title-short}, as measured hourly and aggregated for the time period defined by the *Start date* and *End date*..
```

File: modules/viewing-product-usage.adoc [hunk 11]
```diff
@@ -0,0 +1,45 @@
...
+* CPU units observation date: The date on which the data displayed on this page for the maximum secured CPU units was collected.
+* Node count observation date: The date on which the data displayed on this page for the maximum secured node count was collected.
+
+Data is collected every 5 minutes from the sensors, so there can be a short delay in displaying the current data. To view historical data, you must configure the *Start date* and *End date* and download the data file. The date range is inclusive and depends on your time zone.
```
Comment (mburke5678) [inline modules/viewing-product-usage.adoc:27]:
```suggestion
The sensors collect data every 5 minutes, so there can be a short delay in displaying the current data. To view historical data, you must configure the *Start date* and *End date* and download the data file. The date range is inclusive and depends on your time zone.
```

**Comment by mburke5678** [comment]
@kcarmichael08 A few suggestions for minimalism (or something). Otherwise LGTM

File: modules/viewing-product-usage.adoc (hunk 5 above)
Comment (kcarmichael08) [reply modules/viewing-product-usage.adoc:18]: I meant this page in the UI, but am rewriting to avoid awkwardness.

**Pull request merged**